apps/docs/src/app/core/component-docs/table/examples/table-pagination-example.component.html

```diff
@@ -0,0 +1,34 @@
+<table fd-table>
+    <thead fd-table-header>
+        <tr fd-table-row>
+            <th fd-table-cell>Column Header</th>
+            <th fd-table-cell>Column Header</th>
+            <th fd-table-cell>Column Header</th>
+        </tr>
+    </thead>
+    <tbody fd-table-body>
+        <tr *ngFor="let row of displayedRows" fd-table-row>
+            <td fd-table-cell>
+                <a fd-link href="#">{{ row.column1 }}</a>
+            </td>
+            <td fd-table-cell>{{ row.date }}</td>
+            <td fd-table-cell><fd-icon [glyph]="row.type"></fd-icon></td>
+        </tr>
+    </tbody>
+    <br/>
+    <tfoot fd-table-footer>
+        <button fd-button [fdMenu]="true" [fdMenuTrigger]="itemsPerPageMenu" style="margin-right: 20px">Items Per Page</button>
+        <fd-pagination
+            [totalItems]="totalItems"
+            (pageChangeStart)="newPageClicked($event)"
+            [itemsPerPage]="itemsPerPage"
+            [currentPage]="currentPage"
+        ></fd-pagination>
```

Comment:
Pagination is still weird
![image](https://user-images.githubusercontent.com/26483208/86907086-ceb9a100-c114-11ea-8fb4-3564feb221b2.png)

Reply:
I agree this needs fixing but I'd consider it out of the scope of this PR

Reply:
mentioned in this defect hunting https://github.com/SAP/fundamental-ngx/issues/2777 i'll do this today